Portfolio.html

```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
+            <img class="smallimage2" src="./images/Counter.png" alt="Counter" />
+            <a class="thirdtxt">2015</a>
+            <p class="longtext">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <ul class="tag01">
+            <li class="tag1">html</li>
+            <li class="tag2">css</li>
+            <li class="tag3">javascript</li>
+          </ul>
+
+          <button class="btn btn-project">See Project</button>
+        </div>
+        <!--END OF third CARD-->
+        <div class="singlecard">
+          <img
+            class="portfolioimage"
+            src="./images/Snapshoot-Portfolio4.png"
+            alt="Snapshoot-Portfolio"
+          />
+          <div>
+            <h1 class="second-boldtext">Multi-Post Stories</h1>
+            <a class="firsttxt">CANOPY</a>
+            <img class="smallimage" src="./images/Counter.png" alt="Counter" />
+            <a class="secondtxt">Back End Dev</a>
+            <img class="smallimage2" src="./images/Counter.png" alt="Counter" />
+            <a class="thirdtxt">2015</a>
+            <p class="longtext">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <ul class="tag01">
+            <li class="tag1">html</li>
+            <li class="tag2">css</li>
+            <li class="tag3">javascript</li>
+          </ul>
+
+          <button class="btn btn-project">See Project</button>
+        </div>
+      </section>
+      <!--END OF fourth CARD-->
+
+      <!--About Myself Section-->
+
+      <section id="personal-info">
+        <div class="allinfo">
+          <div>
+            <h1 class="firstintro">About Myself</h1>
+            <p class="secondparagraph">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+
+            <a class="lastskyblue">LET’S CONNECT</a>
+          </div>
+
+          <div class="social-media-icons2">
+            <img src="./images/Github.png" alt="Github" class="githublogo2" />
+            <img
+              src="./images/Linkedin-icon.png"
+              alt="Linkedin"
+              class="linkedinlogo2"
+            />
+            <img
+              src="./images/twofingers.png"
+              alt="twofingers"
+              class="Twofingerslogo2"
+            />
+            <img
+              src="./images/twitter.png"
+              alt="twitter"
+              class="Twitterlogo2"
+            />
+
+            <img src="./images/Mail.png" alt="Mail" class="Maillogo2" />
+          </div>
+
+          <button class="lasttxt2">get my resume</button>
```

Comment:
Please make sure that this button is exactly the same as in the design. Text is not centered both horizontally and vertically. The button does not have a hover and active effect as in the design.

Yours:
![your-btn](https://user-images.githubusercontent.com/35504149/171045444-36891e5c-24e5-477f-9adc-ddc9da092520.png)

Design:

![resume](https://user-images.githubusercontent.com/35504149/171045482-43ad3fae-f316-4a1b-941e-cb763de780ec.png)